### PR TITLE
Typo correction heap_size.asciidoc

### PR DESCRIPTION
**Pull request by bloublou2014**
"-Xms4000mb -Xmx4000mb" on example (Xms/Xmx instead of Xmx/Xmx)

### CODE REVIEW

**Comment by bloublou2014** [comment]
CLA signed

File: docs/reference/setup/sysconfig/heap_size.asciidoc [hunk 1]
```diff
@@ -61,7 +61,7 @@ in the jvm.options file and setting these values via `ES_JAVA_OPTS`:
 [source,sh]
 ------------------
 ES_JAVA_OPTS="-Xms2g -Xmx2g" ./bin/elasticsearch <1>
-ES_JAVA_OPTS="-Xmx4000mb -Xmx4000mb" ./bin/elasticsearch <2>
+ES_JAVA_OPTS="-Xms4000mb -Xmx4000mb" ./bin/elasticsearch <2>
```
Comment (clintongormley) [inline docs/reference/setup/sysconfig/heap_size.asciidoc]: This should actually be:

    ES_JAVA_OPTS="-Xms4000m -Xmx4000m" ./bin/elasticsearch <2>

The JVM doesn't understand `mb`

**Comment by clintongormley** [comment]
Hiya @bloublou2014 

Thanks for the PR.  I've added one comment.  Also, I can't find the signed CLA... any chance you could forward it to me (clinton at elastic dot co)?

**Comment by bloublou2014** [comment]
CLA mail confirmation fwd to clinton at elastic co

**Comment by clintongormley** [comment]
thanks @bloublou2014 - got the CLA.  If you could push a fix for the comment I made, I'll be happy to merge this

**Comment by bloublou2014** [comment]
Change effective for mb to m.

**Comment by clintongormley** [comment]
thanks @bloublou2014

**Pull request merged**